### DISCUSSION TITLE
Real-time monitor (OSD) similar to Armoury Crate's?

### DISCUSSION THREAD

**Discussion by GeoGC** [seerge/g-helper#1791] [Ideas]
Are there any plans to add a similar feature that can show ingame FPS, temperature of CPU/GPU, RAM usage and all that stuff similar to what Armoury Crate has? Or you think it will use too much resources which won't be worth to have on G-Helper?

**Comment by seerge** [comment]
@GeoGC i was already answering same question https://github.com/seerge/g-helper/discussions/1448
